app/assets/javascripts/modules/track-click.js

```diff
@@ -8,16 +8,26 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       element.on('click', trackClick);
 
       var options = {},
-          category = element.attr('data-track-category'),
-          action = element.attr('data-track-action'),
-          label = element.attr('data-track-label');
+          category = element.data('track-category'),
```

Comment:
I think we should use `attr()` instead of `data()` when fetching data from these data attributes, more details here: https://github.com/alphagov/static/commit/ebe69aa77528fd669ff34ce1f093fa45026cd451#diff-5649f4c1766856649d723e2ef9a026b3